js/core/component.js

```diff
@@ -490,6 +456,23 @@ var Component = Class.inherit({
         return result;
     },
 
+    _convertToObj(options, value) {
+        if(typeof options !== "string") return options;
+
+        const result = {};
+        result[options] = value;
+        return result;
+    },
+
+    _getOptionByStealth: function(name) {
+        return this._optionManager.getValueSilently(name);
+    },
+
+    _setOptionByStealth: function(options, value) {
+        options = this._convertToObj(options, value);
+        this._optionManager.setValueSilently(options);
```

Comment:
No need to reassign the attribute:
```suggestion
        this._optionManager.setValueSilently(this._convertToObj(options, value));
```